hazelcast-cloud/src/test/java/com/hazelcast/aws/DescribeInstancesTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.aws;
+
+import com.hazelcast.aws.impl.DescribeInstances;
+import com.hazelcast.config.AwsConfig;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class DescribeInstancesTest {
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testDescribeInstances_whenAwsConfigIsNull() {
+        new DescribeInstances(null,"endpoint");
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testDescribeInstances_whenAccessKeyNull() {
+        new DescribeInstances(new AwsConfig(),"endpoint");
+    }
+
```

Comment:
These are tests for the unhappy situations; which is good to have. But what about the happy situations? Currently we have no idea if that is working.

PS:
Your test is called DescribeInstances; so you are already indicating that you are testing this. That is why you don't need to repeat it in the test name.

So it can be simplified to:
test_whenAwsConfigIsNull

Or if you want to be even shorter:
whenAwsConfigIsNull

Also make sure you do a code reformat.